fix: typo (#22)
CeresDB/docs: docs/src/en/wal_on_kafka.md

│                                    │
└────────────────────────────────────┘
```
### Write and read logs
The writing and reading process in a region is simple. 

For writing:
+ Open the specified region (auto create it if necessary).
+ Put the logs to specified Kafka partition by client.
+ Update `next_seq_num`,  `current_high_watermark` and `seq_offset_mapping` in corresponding `TableMeta`. 

For reading:
+ Open the specified region.
+ Just read all the logs of the region, and the split and replay work will be done by the caller.

### Delete logs

> Log's deletion can be split to two steps:

Reviewer: ```suggestion
Log deletion can be divided into two steps:
```
